File: drools/drools-examples/src/main/org/drools/examples/fibonacci/Fibonacci.java
package org.drools.examples.fibonacci;

public class Fibonacci
{
    private int sequence;
    private int value;

    public Fibonacci(int sequence)
    {
        this.sequence = sequence;
        this.value    = -1;
    }

    public int getSequence()
    {
        return this.sequence;
    }

    public void setValue(int value)
    {
        this.value = value;
    }

    public int getValue()
    {
        return this.value;
    }

    public String toString()
    {
        return "Fibonacci(" + this.sequence + "/" + this.value + ")";
    }
}

File: trunk/drools/drools-dotnet/src/semantics.dotnet.tests/examples/fibonacci/Fibonacci.cs
using System;

namespace org.drools.semantics.dotnet.tests.examples.fibonacci
{
	public class Fibonacci
	{
		private int _sequence;
		private long _value;

		public Fibonacci(int sequence)
		{
			_sequence = sequence;
			_value = -1;
		}

		public virtual int Sequence
		{
			get{ return _sequence; }	
		}

		public virtual long Value
		{
			get { return _value; }
			set{ _value = value; }			
		}

		public override string ToString()
		{
			return "Fibonacci(" + _sequence + "/" + _value + ")";
		}
	}
}